packages/nitro-protocol/docs/state-channels/overview.md

```diff
@@ -0,0 +1,74 @@
+---
+id: overview
+title: Overview
+---
+Welcome to the nitro-protocol state channels documentation site. In this introductory pages, you will find a high level description of state channels; their use-cases and the challenges they represent; our solution to those challenges. You will also find a quick start guide to help you nitro-charge / channelize your DApp using our framework.
+
+Elsewhere on the site you will find API documentation for a reference implementation of nitro that is gas optimized and tested.
+
+## What is a State Channel?
+
+State channels are a technique improving the scalability of a blockchain, and for increasing throughput for some of its Decentralized Applications (DApps).
+
+A state channel provides an environment where a set of parties can update a shared off-chain state, by exchanging signed data between themselves.
+Once the parties have finished, they may present the final version of the state to the blockchain, which in turn triggers changes to the on-chain state.
+
+The state channel leverages properties of the underlying chain, to allow the entire operation to be conducted without the need for the parties to trust one another.
+
+### A simple example
+
+The quickest way to gain an understanding of state channel and their capabilities is by looking at a simple example.
+Here we look at the example of a two-party, _direct_ state channel, between Alice and Bob.
+
+Let's suppose that Alice and Bob want to use a state channel for payments: for example, Alice might be streaming a video from Bob, and wants to pay Bob per megabyte she consumes.
+Executing each payment as a blockchain transaction would be too slow and costly.
+
+To execute the payments through a state channel, instead, Alice and Bob proceed as follows:
+
+1. Alice and Bob agree to open a state channel. By doing this, they agree to a `channelId`, an initial state, and a set of update rules. In this case, the initial balances might be `{A: 10, B: 0}` - 10 coins to Alice and 0 coins to Bob. The update rules might say that Alice can send coins to Bob, but Bob can't unilaterally take coins from Alice.
+2. Alice deposits the coins into an _asset holder_, where they are held specifically for the Alice and Bob's channel. In our example, Bob doesn't need to send any coins because his channel balance starts out at 0 coins.
```

Comment:
Grammer:
"... where they are held specifically for ~the~ Alice and Bob's channel"